config/locales/server.es.yml

```diff
@@ -1236,6 +1236,7 @@ es:
     pop3_polling_host: "El host utilizado para hacer polling de e-mails vía POP3."
     pop3_polling_username: "El nombre de usuario de la cuenta POP3 para hacer polling de e-mails."
     pop3_polling_password: "La contraseña de la cuenta POP3 para hacer polling de correos."
+    pop3_polling_delete_from_server: "Eliminar los e-mails del servidor"
```

Comment:
There is no need to provide translations other than English because they will be overwritten next time we pull translations from Transifex.